app/views/notifications/_comment.html.erb

```diff
@@ -0,0 +1,8 @@
+<% commentable = comment.commentable %>
+<% if commentable.title? %>
+  <%= presenter.verb %> on the
```

Comment:
This verb should be `commented` or `added a comment` for comments?
<img width="569" alt="captura de pantalla 2018-07-16 a las 10 59 37" src="https://user-images.githubusercontent.com/85766/42752992-752e9f26-88e7-11e8-8b03-375c59643024.png">

Reply:
Also the link may point to the exact comment https://github.com/dradis/dradis-ce/blob/comments-feature/app/controllers/comments_controller.rb#L11